### PR TITLE
testing/libexosip2: new aport

### PR DESCRIPTION
**Pull request by dyfet**
This includes libexosip2 and libosip2 as a dependency and also a new aport:

    testing/libexosip2: new aport
    
    http://savannah.nongnu.org/projects/exosip
    Extended osip2 library

    testing/libosip2: new aport
    
    https://www.gnu.org/software/osip
    oSIP is an implementation of SIP

### CODE REVIEW

File: testing/libosip2/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+license="LGPL-2.1-or-later"
+makedepends="
+	autoconf
+	automake
+	libtool
+"
+options="libtool"
+subpackages="$pkgname-dev $pkgname-doc"
+source="
+	http://ftp.gnu.org/gnu/osip/$pkgname-$pkgver.tar.gz
```
Comment (Cogitri) [inline testing/libosip2/APKBUILD]: Please don't use $pkgname in source URLs

File: testing/libosip2/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	"
```
Comment (Cogitri) [inline testing/libosip2/APKBUILD]: super nitty, but please leave a newline here.

Comment (dyfet) [reply testing/libosip2/APKBUILD]: I took some time to look at other packages.  I used to have a patch and didn't need that anymore, so I actually should have consolidated the source line back into one line.  I also now see other packages do indeed typically use name-$pkgvar.tar.gz in source url's.  Especially to do my first packages, it did help me to compare more carefully with existing packages.

Comment (Cogitri) [reply testing/libosip2/APKBUILD]: I can recommend you to use `apkbuild-lint` from the `atools ` package which will spot these things for you :)

Comment (dyfet) [reply testing/libosip2/APKBUILD]: Actually apkbuild-lint returned no warnings...so it did not check for that.   I originally had a bunch of cd $builddir which it did warn me about, and a few other minor things, though.

**Comment by Cogitri** [comment]
Thanks. For the future: Please just amend these changes to the respective commit :)

**Pull request closed without merging**